lints/etsi/lint_qcstatem_psd2_psd2statem_encoding.go

```diff
@@ -0,0 +1,54 @@
+package etsi
+
+/*
+ * ZLint Copyright 2020 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/lint"
+	"github.com/zmap/zlint/util"
+)
+
+type qcStatemPsd2Pd2StatemEnc struct{}
+
+func (l *qcStatemPsd2Pd2StatemEnc) Initialize() error {
+	return nil
+}
+
+func (l *qcStatemPsd2Pd2StatemEnc) CheckApplies(c *x509.Certificate) bool {
+	if !util.IsExtInCert(c, util.QcStateOid) {
```

Comment:
I tried leaving this comment yesterday but Github was having issues

* Don't abbreviate as State, since that's a different word
* Be consistent, whatever you choose
* There's probably nothing wrong with just spelling out Statement, we're not capping lines to 80 or 100 characters.